lib/knapsack/distributors/report_distributor.rb

```diff
@@ -20,73 +20,49 @@ def node_time_execution
       private
 
       def post_assign_test_files_to_node
-        assign_slow_test_files
-        assign_remaining_test_files
+        assign_test_files
         sort_assigned_test_files
       end
 
       def sort_assigned_test_files
-        ci_node_total.times do |index|
-          # sort by first key (file name)
-          # reverse it and then sort by second key (time) in reverse order
-          node_tests[index][:test_files_with_time].sort!.reverse!.sort! do |x, y|
-            y[1] <=> x[1]
-          end
+        node_tests.map do |node|
+          node[:test_files_with_time].sort_by! { |file_name, _time| -file_name }
```

Comment:
I will keep `reverse!` because sorting `file_name` string with `-` char does not work in Ruby 2.2 https://github.com/KnapsackPro/knapsack/runs/2896841914?check_suite_focus=true